docs/apps/lnw/es2k/es2k-linux-networking.md

```diff
@@ -86,37 +83,38 @@ Packets coming from overlay network:
     ovs-vsctl add-port br-int <Physical port PR>
     ```
 
-##### Egress traffic with VxLAN encap
+##### Egress traffic with VxLAN encapsulation
 
 Packets coming from overlay network:
 
 - Determine the source port of the packet based on which overlay VSI the packet has landed on.
 - Validate if the source port is part of the bridge, else drop the packet.
 - If valid bridge configuration is found, find the PR associated with the bridge and forward the packet to the PR in ACC.
-- OvS control plane receives the packet and forwards the packets to the destined VxLAN port if MAC is already learnt, else flood the packet in the respective bridge.
+- OvS control plane receives the packet and forwards the packet to the destined VxLAN port if MAC is already learnt, else flood the packet in the valid bridge found.
 - Once the packet reaches the VxLAN port, here the kernel checks the routing table to reach `remote_ip` that is configured for the OvS VxLAN tunnel.
-- Underlay network to reach `remote_ip` is configured on a TEP termination bridge. Here, the kernel resolves ARP of the underlay network.
-- Once ARP is resolved, kernel encapsulates the packet and this packet will be forwarded to the destined PR of the physical port if MAC is already learnt, else flood the packet in the respective TEP termination bridge.
+- Underlay network to reach `remote_ip` is configured on a TEP termination bridge. The kernel resolves the ARP for underlay network.
```

Comment:
```suggestion
- Underlay network to reach `remote_ip` is configured on a TEP termination bridge. The kernel resolves the ARP table for underlay network.
```
I think we need a ARP table spec there? `resolves the ARP` doesnt sound right

Reply:
ARP resolution is standard networking terminology. Once the ARP is resolved (where IP address is associated with the physical MAC address), the table is updated with the resolved entry